### DISCUSSION TITLE
List of available samplers for --sample_sampler

### DISCUSSION THREAD

**Discussion by thr3a** [kohya-ss/sd-scripts#268] [Q&A]
The sample image generation feature during training is very useful.
I found a list of samplers that can be specified with "--sample_sampler" in the source code, but I don't understand the correspondences.

https://github.com/kohya-ss/sd-scripts/blob/45945f698a7c57b83564e7ad3845c7b677899a5b/library/train_util.py#L2242

I want to use the DPM++ 2M Karras or DPM++ SDE Karras sampler. What values should I specify for that?

**Comment by mercpls** [comment]
I'm also looking to use `DPM++ SDE Karras sampler`
